themes/powerline-multiline/powerline-multiline.theme.bash

```diff
@@ -49,6 +49,9 @@ RBENV_THEME_PROMPT_SUFFIX=""
 RUBY_THEME_PROMPT_COLOR=${POWERLINE_RUBY_COLOR:=161}
 RUBY_CHAR=${POWERLINE_RUBY_CHAR:="❲r❳ "}
 
+TERRAFORM_THEME_PROMPT_COLOR=${POWERLINE_TERRAFORM_COLOR:=161}
+TERRAFORM_CHAR=${POWERLINE_TERRAFORM_CHAR:="❲t❳ "}
+
```

Comment:
For consistency, we should place new configurations in the same place in each of the powerline theme files.

This way, you can do a `diff` of each file and see only  the meaningful differences between the files.

In this file, its your new configurations are above Kubernetes, but in other files its below Kubernetes.

I don't have a preference either way, as longs its consistent in all files.

Reply:
good point, I have fixed the order